deploy/sagemaker.yaml

```diff
@@ -3,11 +3,18 @@ Resources:
   BlueoilNotebook:
     Type: 'AWS::SageMaker::NotebookInstance'
     Properties:
-      InstanceType: "ml.t2.medium"
+      InstanceType: "ml.t3.medium"
       NotebookInstanceName: "BlueoilNotebook"
+      LifecycleConfigName: !GetAtt NotebookLifecycleConfig.NotebookInstanceLifecycleConfigName
       RoleArn:
         Fn::Join:
           - ':'
           - - 'arn:aws:iam:'
             - Ref: 'AWS::AccountId'
             - role/BlueoilSagemakerRole
+  NotebookLifecycleConfig:
+    Type: 'AWS::SageMaker::NotebookInstanceLifecycleConfig'
+    Properties:
+      NotebookInstanceLifecycleConfigName: clone-bluegrass-repository
+      OnCreate:
+        - Content: 'IyEvYmluL2Jhc2gKc3VkbyAtdSBlYzItdXNlciAtaSA8PCdFT0YnCmNkIC9ob21lL2VjMi11c2VyL1NhZ2VNYWtlciAmJiBnaXQgY2xvbmUgaHR0cHM6Ly9naXRodWIuY29tL0xlYXBNaW5kL2JsdWVncmFzcy5naXQKRU9G'
```

Comment:
@hariby 
I think this setting is for cloning our repository as default.
How about using `DefaultCodeRepository` to clone our bluegrass repository? I think it is enough and suitable for this use.
https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-sagemaker-notebookinstance.html#cfn-sagemaker-notebookinstance-defaultcoderepository